x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -16,7 +16,7 @@ export const CREDENTIALS_PATH = '/credentials';
 export const ROLE_MAPPINGS_PATH = '#/role-mappings'; // This page seems to 404 if the # isn't included
 
 export const ENGINES_PATH = '/engines';
-export const CREATE_ENGINES_PATH = `${ENGINES_PATH}/new`;
+export const CREATE_ENGINES_PATH = `/create_engine`;
```

Comment:
@zumwalt is good with this change https://github.com/elastic/kibana/pull/89156#issuecomment-768493626